services/terrain/src/terrain/services/permanent_id_requests.clj

```diff
@@ -259,16 +259,21 @@
           {:keys [irods-avus metadata]} (metadata-get user folder-id)
           ezid-metadata                 (parse-valid-ezid-metadata irods-avus metadata)
           ezid-response                 (ezid/mint-id shoulder ezid-metadata)
+          identifier                    (get ezid-response (keyword type))
           publish-path                  (publish-data-item user folder (ezid-response->avus ezid-response))]
-      (email/send-permanent-id-request-complete type publish-path))
+      (email/send-permanent-id-request-complete type
+                                                publish-path
+                                                (json/encode ezid-response {:pretty true}))
```

Comment:
I guess that the `identifiers` parameter in the email is this json-encoded response? What'll be in that response, other than the identifiers themselves (and should we perhaps update the variable name the template uses, so it's clearer it'll be more than just identifiers)?